config/config.yml

```diff
@@ -22,7 +22,7 @@ defat-config:
     certKey:
     caFile:
   sign-key:
-  users-file: /home/ubuntu/defat/config/users.yml
+  users-file: /home/ubuntu/vlad/sec03/defatt/config/users.yml
 
 docker-repositories:
```

Comment:
This `docker-repositories` section is private and should not be pushed to git in any case. I have revoked the credentials here however, it is good practice to do not include credentials to git which are in use.

Reply:
Agreed! 
Is there a way to ignore certain lines, or what is the approach in certain situations?

Reply:
Not certain lines however it is ideal case to add config.yml file to [.gitigonore](https://github.com/aau-network-security/defatt/blob/1fb42939dc03971ea3c81a7b549f72025f4c3678/.gitignore#L17)

Reply:
This might be useful for the config file https://pkg.go.dev/os?utm_source=godoc#UserConfigDir

Reply:
If you have time @Lanestolen try to make a pull request with it.